packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -6,11 +6,21 @@ import 'package:flutter/animation.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
-// Internal representation of a child that, now or in the past, was set on the
-// PageTransitionSwitcher.child field, but is now in the process of
-// transitioning. The internal representation includes fields that we don't want
-// to expose to the public API (like the controllers).
+/// An internal representation of a child widget subtree that, now or in the past,
+/// was set on the [PageTransitionSwitcher.child] field and is now in the process of
+/// transitioning.
+///
+/// This class should not be used outside of the context of
+/// [PageTransitionSwitcher.layoutBuilder], which uses this class in a callback to
+/// customize the layout of the transitioning widgets.
+///
+/// For more information, please see the documentation of
+/// [PageTransitionSwitcher.layoutBuilder].
```

Comment:
This comment seems outdated. It's not used as part of the layoutBuilder anymore?